src/wasm/syscalls_env.cpp

```diff
@@ -134,12 +134,12 @@ namespace wasm {
         // Fake system info
         // TODO - should probably give some valid stuff here in case we break something
         wasm_utsname s{
-                .sysname="Linux",
-                .nodename="faasm",
-                .release="1.0.0",
-                .version="Faasm 123",
-                .machine="x86",   // Probably safest in 32-bit wasm env
-                .domainname="(none)"
+                { .sysname="Linux" },
+                { .nodename="faasm" },
+                { .release="1.0.0" },
+                { .version="Faasm 123" },
+                { .machine="x86" },   // Probably safest in 32-bit wasm env
+                {.domainname="(none)" }
```

Comment:
Was this really a compiler warning? I'm surprised as the other syntax doesn't seem that offensive to me.

Reply:
I doubled checked: https://stackoverflow.com/questions/18731707/why-does-c11-not-support-designated-initializer-lists-as-c99